robottelo/vm.py

```diff
@@ -162,10 +161,9 @@ def create(self):
                 u'Failed to run snap-guest: {0}'.format(result.stderr))
 
         # Give some time to machine boot
-        time.sleep(60)
-
         result = ssh.command(
-            u'ping -c 1 {0}.local'.format(self._target_image),
+            u'for i in {{1..60}}; do ping -c1 {0}.local && break; sleep 1;'
+            u' done; echo $?'.format(self._target_image),
```

Comment:
This could help may be, if we face any issue. https://www.freedesktop.org/wiki/Software/systemd/NetworkTarget/